browser/css/notebookbar.css

```diff
@@ -107,6 +107,9 @@
 	margin-left: 11px;
 }
 
+#Save1.savemodified {
```

Comment:
`.unoSave` class could be used instead of `Save1`, `1` is generated to make ids unique but it's not guaranted to be in the same place, also might be changed in the future by some side effects of other changes...

Reply:
Yes we have this problem still in more than on place, but thanks for the reminder, I think better to don;t include yet another bad practice here